src/config/section/storage.js

```diff
@@ -163,7 +163,12 @@ export default {
           icon: 'delete',
           label: 'Delete Volume',
           dataView: true,
-          groupAction: true
+          groupAction: true,
+          show: (record, store) => { // g_allowUserExpungeRecoverVolume
```

Comment:
@shwstppr can you explain the comment, do we need to check anything else? Shouldn't we allow users to delete a volume?

Reply:
@rhtyd just checked g_allowUserExpungeRecoverVolume is value from capabilities, `json.listcapabilitiesresponse.capability.allowuserexpungerecovervolume`. How to access that in Primate if set globally?

For users, role based check is an OR condition so it will show up for user
![Screenshot from 2020-03-27 17-02-56](https://user-images.githubusercontent.com/153340/77752056-d618db00-704c-11ea-8ebc-ad6925a2136c.png)